mesh/mesh.go

```diff
@@ -500,3 +500,81 @@ func (m *Mesh) GetATXs(atxIds []types.AtxId) (map[types.AtxId]*types.ActivationT
 	}
 	return atxs, mIds
 }
+
+// ActiveSetForLayerConsensusView - returns the active set size that matches the view of the contextually valid blocks in the provided layer
+func (m *Mesh) ActiveSetForLayerConsensusView(layer types.LayerID, layersPerEpoch uint16) (map[string]types.AtxId, error) {
```

Comment:
why are you returning the atxids? the hare only uses the miner ids

Reply:
Fixed